modules/develop/partials/run-transforms.adoc

```diff
@@ -245,9 +249,117 @@ func filterValidJson(e transform.WriteEvent, w transform.RecordWriter) error {
 	return nil
 }
 ```
-
+====
 --
+Rust::
++
+--
+. Create and initialize a data transforms project:
++
+```bash
+rpk transform init --language=rust
+```
++
+If you do not include the `--language` flag, the `transform init` command will prompt you for the language.
++
+A successful command generates project files in your current directory:
++
+[.no-copy]
+----
+.
+├── Cargo.lock
+├── Cargo.toml
+├── README.md
+├── src
+│  └── main.rs
+└── transform.yaml
+----
+The `src/main.rs` file contains the transform logic, and the `transform.yaml` file specifies the transform's configuration.
++
+TIP: When creating a custom data transform, initialization steps can be done in `main.rs` because it's only run once at the start of the package. Although state can be cached in global variables, Redpanda may restart a Wasm module at any point, which causes the state to be lost.
```

Comment:
```suggestion
TIP: When creating a custom data transform, initialization steps can be done in `main()` because it's only run once at the start of the package. Although state can be cached in global variables, Redpanda may restart a Wasm module at any point, which causes the state to be lost.
```

It's the main function, not really just the main file.